Hi3Helper.Core/Lang/en.json

```diff
@@ -73,8 +73,8 @@
     "RegionLoadingTitle": "Loading Region",
     "RegionLoadingAPITitle1": "Loading",
     "RegionLoadingAPITitle2": "Launcher API",
-    "RegionLoadingSubtitleTimeOut": "Loading Region {0} has timed-out (> {1} seconds). Retrying...",
-    "RegionLoadingSubtitleTooLong": "Retry is taking a bit longer than expected. Please ensure that your Internet connection is stable!",
+    "RegionLoadingSubtitleTimeOut": "Action timed out ({1}). Retrying to load Region {0}...",
```

Comment:
![image](https://user-images.githubusercontent.com/28079733/229973273-3327f40d-48e0-4608-b3d9-84da985a3b61.png)
Re-add `(> {1} seconds)` so it won't just say `15` in the output.

Reply:
Fixed.